meilisearch/tests/__init__.py

```diff
@@ -1,2 +1,8 @@
 MASTER_KEY = 'masterKey'
 BASE_URL = 'http://127.0.0.1:7700'
+
+def clear_all_indexes(client):
+    indexes = client.get_indexes()
+    uids = [index['uid'] for index in indexes]
+    for uid in uids:
+        client.get_index(uid).delete()
```

Comment:
```suggestion
    indexes = client.get_indexes()
    for index in indexes:
        client.get_index(index['uid']).delete()
```

Reply:
```suggestion
    indexes = client.get_indexes()
    for index in indexes:
        resp = client.get_index(index['uid']).delete()
        if resp.status_code != 204:
            raise Exception("Test environment couldn't be cleaned properly")
```

Reply:
I propose two suggestions:

1. We don't need to store the`uids` in a list. We can delete the index directly.
2. We can check that the deletion was succesful, because if there is a problem it means that the environment wasn't clean before running the tests

:)

Reply:
If there is another response that a 204 (= if an index is not deleted properly) an exception is already raised with my suggestion.
I would rather have the status code and the message raised (thanks to our new error handler 😎) than `Test environment couldn't be cleaned properly` without any other information 😉

But I like your first suggestion!!

Reply:
You are right, it will indeed raise an exception, so this check would be redundant and unnecessary !